inst/htmlwidgets/plugins/Leaflet-MiniMap/Minimap-binding.js

```diff
@@ -1,11 +1,24 @@
-  LeafletWidget.methods.addMiniMap = function(position, width, height , collapsedWidth,
```

Comment:
Let's change DESCRIPTION to 1.0.2.9000, and create a new 1.1 item in NEWS. When you believe you have a build that's ready for CRAN, then change to 1.1 in DESCRIPTION.